doc/articles/features/fluent-styles.md

```diff
@@ -0,0 +1,34 @@
+# Fluent-styled controls
+
+Uno 3.0 and above supports control styles conforming to the [Fluent design system](https://www.microsoft.com/design/fluent). This article explains how to use them in your app.
+
+## Using Fluent styles in a newly-created app
+
+If you have the [latest version of the Uno app template](https://marketplace.visualstudio.com/items?itemName=nventivecorp.uno-platform-addin) installed, there's nothing to do - newly-created Uno apps will use Fluent control styles by default on all platforms.
+
+## Upgrading existing Uno apps to use Fluent styles
+
+To modify an existing Uno app to use the Fluent control styles, follow these steps:
+
+1. Update `Uno.UI` NuGet packages to 3.0 or above.
+1. In the `UWP` head project of your solution, install the [WinUI 2 NuGet package](https://www.nuget.org/packages/Microsoft.UI.Xaml).
```

Comment:
What if the solution does not include the UWP head project? It's my understanding (and confirmed in my own testing of a small app) that it's possible to build an Uno app using only one project head (Droid for example) and a customized shared project. It's not clear to me how WinUI is being consumed by Uno here but I hope it would be possible to use FluentUI styles even without a UWP head.

Reply:
Yes, this is only to make the UWP head itself build, it doesn't affect any other targets.

This doc was already merged separately, feel free to edit and clarify :) https://github.com/unoplatform/uno/blob/master/doc/articles/features/fluent-styles.md

Reply:
@davidjohnoliver I can make the updates. It's still not entirely clear to me how Fluent assets are enabled in Uno though. For UWP apps the article simply follows the UWP/WinUI 2.x approach. But how is this handled in Uno? My assumption was Uno 3.0+ would default to Fluent styles all the time. However, there is the also the checkbox in the sample app. What is the mechanism for making this work? I think that needs to be documented as well.

I think breaking this document up into two sections makes the most sense:
 1. Fluent styling for Xamarin-based Projects
 2. Fluent styling for UWP Projects (Following WinUI 2.x instructions)

Reply:
@robloo They're handled the same way as for WinUI 2 (and 3): they are implicit styles defined in `XamlControlsResources`, and including them in your `App.Resources` (step 3) makes them the default for the app. The only difference from UWP is that installing an additional NuGet package (step 2) is not necessary, because we bundle supported WinUI 2 features directly with Uno.UI. (Which includes `XamlControlsResources` containing the Fluent style.)

Reply:
Got it! Thanks for the clarification.

Reply:
Added PR #4070 to address this conversation